change/msal-2020-08-12-14-47-04-b2c-multiple-policies.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "B2C Multiple Policy Support (#1757)",
+  "packageName": "msal",
+  "email": "thomas.l.norling@gmail.com",
```

Comment:
Wondering if this should be a microsoft email?

Reply:
Yes! Thanks for catching this!